scripts/modules/elastic_stack.py

```diff
@@ -153,8 +162,10 @@ def add_es_config(args, prefix="output"):
                     args.append((prefix + ".elasticsearch.{}".format(cfg), self.options[es_opt]))
                 elif self.options.get("xpack_secure"):
                     args.append((prefix + ".elasticsearch.{}".format(cfg), default_apm_server_creds.get(cfg)))
+            if tls:
+                args.append((prefix + ".elasticsearch.ssl.certificate_authorities", "['" + self.STACK_CA_PATH + "']"))
 
-        es_urls = self.options.get("apm_server_elasticsearch_urls") or [self.DEFAULT_ELASTICSEARCH_HOSTS]
```

Comment:
question: do we need to update the `self. DEFAULT_ELASTICSEARCH_HOSTS` usage across different services (like aux_services, opbeans and beats? They still use the variable

Reply:
I gues so.

Reply:
yes certainly.  We also need to apply these tls client settings to all beats, I'd prefer that be done in a follow up

Reply:
if you do not have time open and issue and we take care of it :D